plugins/ssh-plugin/src/ssh-plugin-backend.ts

```diff
@@ -17,9 +17,21 @@ import { pathExists, unlink, ensureFile, chmod, readFile, writeFile, appendFile
 import * as os from 'os';
 import { accessSync, mkdtempSync, readFileSync, rmdirSync, unlinkSync } from 'fs';
 import { R_OK } from 'constants';
+import { spawn } from 'child_process';
 
 export async function start(): Promise<void> {
 
+    const sshKeyManager = new RemoteSshKeyManager();
+    let keys: cheApi.ssh.SshPair[] = [];
+    try {
+        keys = await getKeys(sshKeyManager);
+    } catch (e) {
+        // No SSH key pair has been defined, do nothing.
```

Comment:
Using exceptions for valid return values (empty/undefined) is generally frowned upon. I know you did not introduce it here, but still.

Reply:
fixed